evap/results/tests/test_views.py

```diff
@@ -52,6 +52,19 @@ def test_multiple_evaluations_per_course(self):
         self.assertNotContains(page, evaluation2.full_name)
         caches['results'].clear()
 
+
+    @patch('evap.evaluation.models.Evaluation.can_be_seen_by', new=(lambda self, user: True))
+    def test_order(self):
+        student = baker.make(UserProfile, email="student@institution.example.com")
```

Comment:
The student doesn't need this email, right?

Reply:
If I don't set this email, the call to `self.app.get` fails since email is nullable and not set automatically. Then, django_webtest does not serialize the user correctly